postcss.config.js

```diff
@@ -16,7 +21,17 @@ module.exports = {
             },
         },
         'postcss-flexbugs-fixes': {},
-        // 'postcss-rtl': {},
-        // cssnano: {},
+        'postcss-rtl': {},
+        'postcss-modules': {
```

Comment:
We can add `generateScopedName` to get names like `d2ui-component__className__hash` if we want to customize the generated names.